src/group/controller/collection.ts

```diff
@@ -11,11 +11,11 @@ class GroupMemberCollectionControlller extends Controller {
     const user = await userService.findById(parseInt(ctx.state.params.id, 10));
 
     /**
-     * Cheks if user type is a group
+     * Checks if user type is a group
      */
-    const groupUser = await groupService.isGroup(user);
+    const group = await groupService.isGroup(user);
```

Comment:
`group` still kinda suggests that the variable is a group itself, not a boolean. I would go for `isGroup`, or better yet... put this function call in the if statement itself. It also won't need `await` anymore